adapter/internal/operator/apis/dp/v1alpha1/backend_types.go

```diff
@@ -61,6 +61,14 @@ type BackendSpec struct {
 
 	// +optional
 	Security []SecurityConfig `json:"security,omitempty"`
+	Retry    *RetryConfig     `json:"retry,omitempty"`
+}
+
+// RetryConfig defines retry configurations
+type RetryConfig struct {
+	MaxRetryCount        uint32   `json:"maxRetryCount,omitempty"`
```

Comment:
set default values

Reply:
Default behavior is handled by envoy itself. Do we really need to load the defaults? If someone wants to customize these values, they can simply add the configs.